src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -169,6 +170,11 @@ public class Iso20022FormatExtractor {
 
     private static final int VENDOR_NUM_MAX_LENGTH = 20;
 
+    /*
+     * CU Customization: Added a helper regex for replacing newlines.
+     */
+    private static final Pattern NEWLINE_PATTERN = Pattern.compile("(\\r\\n|\\r|\\n)");
```

Comment:
Should the string be moved to a a constant? I am fine with the way it is if that is simpler for readability.

Reply:
We usually don't create a separate constant for a static regex string, especially if it's only used in one spot. This static field is already declared as a constant and the `Pattern` class is immutable, so I would prefer to keep this definition as-is.